viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -33,9 +33,15 @@ import {
   PointerEventData,
   SceneHandler
 } from '@reveal/utilities';
-import { CameraManager, ProxyCameraManager, StationaryCameraManager } from '@reveal/camera-manager';
+import {
+  CameraManager,
+  FlexibleCameraManager,
+  ProxyCameraManager,
+  StationaryCameraManager
+} from '@reveal/camera-manager';
 import { MetricsLogger } from '@reveal/metrics';
 import debounce from 'lodash/debounce';
+import { moveCameraPositionTo, tweenCameraToDefaultFov } from '@reveal/camera-manager/src/Flexible/moveCamera';
```

Comment:
`moveCameraPositionTo` and `tweenCameraToDefaultFov` should be exported within `camera-manager` package and imported as `from '@reveaal/camera-manager`

Reply:
I move these functions in the end of the file instead